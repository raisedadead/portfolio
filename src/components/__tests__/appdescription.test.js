import AppDescription from '../appdescription'
import React from 'react'
import renderer from 'react-test-renderer'

describe(`AppDescription`, () =>
  it(`renders correctly -- according to react-test-renderer`, () => {
    const tree = renderer.create(<AppDescription />).toJSON()
    expect(tree).toMatchSnapshot()
  }),
)
