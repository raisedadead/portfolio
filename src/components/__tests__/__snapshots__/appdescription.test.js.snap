// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppDescription renders correctly -- according to react-test-renderer 1`] = `
<section>
  <p
    data-testid="app-description"
  >
    This is a Gatsbyjs starter that I use to test things and practice.  You are welcome to look around for examples how to use many Gatsby plugins and utilities.
  </p>
  <code
    className="block"
  >
    $ gatsby new my-website https://github.com/DavidSabine/gatsby-starter
    <br />
    # these commands presume that you have node, yarn, git, and gatsby-cli
    <br />
    # visit the readme at 
    <a
      href="//github.com/DavidSabine/gatsby-starter/blob/master/README.md"
      onClick={[Function]}
    >
      github.com/DavidSabine/gatsby-starter
    </a>
  </code>
  <p>
    To see it all in action, then run the following:
  </p>
  <code
    className="block"
  >
    $ yarn
    <br />
    $ yarn develop # then go to http://localhost:8000 in your browser
    <br />
  </code>
  <p>
    To see the tests run, use...
  </p>
  <code
    className="block"
  >
    $ yarn test
    <br />
    $ yarn test:e2e
    <br />
  </code>
  <p>
    The demo of this starter is at 
    <a
      href="//gatsby-starter.davesabine.com"
      onClick={[Function]}
    >
      gatsby-starter.davesabine.com
    </a>
     and the full source code is at 
    <a
      href="//github.com/DavidSabine/gatsby-starter"
      onClick={[Function]}
    >
      github.com/DavidSabine/gatsby-starter
    </a>
    .
  </p>
</section>
`;
